styles/main.css

```
@@ -0,0 +1,80 @@
+body {
+    background-color: #f1f2f4;
+}
+
+.categories{
+    min-width: 6rem;
+}
+
+.categories>img {
+    width: 4rem;
+}
+
+.categories span {
+    font-size: 0.9rem;
+    white-space: nowrap;
+}
+
+.dropdown-svg {
+    width: 1.2rem;
+}
+
+.product-section-1 {
+    height: 22rem;
+}
+
+.product-section-2 {
+    max-width: 100%;
+}
+
+.announcement {
+    min-height: 4rem;
+}
+
+.announcement-1 {
+    min-height: 4.5rem;
+}
+
+.announcement span, .announcement-1 span {
+    white-space: nowrap;
+}
+.offer-card {
+    cursor: pointer;
+}
+
+.clock-img {
+    width: 1.5rem;
+}
+
+.product-feed-1 {
+    width: 100%;
+}
+
+.product-feed-1>a {
+    min-width: 12rem;
+}
+
+.product-feed-1 p {
+    width: 100%;
+    text-align: center;
+    white-space: nowrap;
+    overflow: hidden;       
+    text-overflow: ellipsis;
+}
+
+.product-feed-1>a p:nth-of-type(3) {
+    color: #999999;
+}
+
+.product-feed-image {
+        width: 10rem;
+        height: 10rem;
```

You can fix the indentation here.
```suggestion
    width: 10rem;
    height: 10rem;
```

---

Removed extra tab.

https://github.com/krushna-simform/flipkart-skeleton-ui/pull/5/commits/d99b8f30b2437638b6cfcfc1bba5d85a37abd1f0